Source/ThirdParty/ik/CMakeLists.txt

```diff
@@ -19,11 +19,16 @@
 # THE SOFTWARE.
 #
 
+include (CheckIncludeFiles)
+
 # Define target name
 set (TARGET_NAME ik)
 
-include (CheckIncludeFiles)
-check_include_files (stdint.h HAVE_STDINT_H)
+check_include_files (stdint.h IK_HAVE_STDINT_H)
```

Comment:
There are two ways to pass the variable from build system to the C++ compiler:

1. def/undef a preprocessor variable either using `cmakedefine` command or  `add_definitions()` conditionally in CMake build script, then using `ifdef / ifndef` in C/C++ code.

2. always define the variable but set its value to 0/1 conditionally, then using `if` directive in code.

I believe Urho prefers the first way.